src/main/java/seedu/duke/parser/Parser.java

```diff
@@ -354,9 +354,13 @@ private Command prepareBye(String arguments) throws IllegalFormatException {
      * Parses report command arguments.
      *
      * @param arguments The additional arguments after command word.
+<<<<<<< HEAD
+     * @return Report object
+=======
      *
      * @return TotalCostAndIncomeCommand object
      *
+>>>>>>> master
```

Comment:
Delete accordingly

Reply:
Thanks for pointing this out, missed this cos wasn't returning any error